reactive/sonarr.py

```diff
@@ -51,10 +65,13 @@ def install_sonarr():
 
 ''')
     apt_update()
-    apt_install('nzbdrone')
+    dependencies = sh.deps
+    dependencies.append('nzbdrone')
+    apt_install(dependencies)
     os.chmod('/opt/', 0o777)
     shutil.chown('/opt/NzbDrone', user=sh.user, group=sh.user)
     host.chownr('/opt/NzbDrone', owner=sh.user, group=sh.user)
+    host.mkdir(sh.home_dir, owner=sh.user, group=sh.user, perms=0o755)
```

Comment:
`0o750` should be OK here, right?

Reply:
Yea that should be fine I think. I'll swap it